containers/client.py

```diff
@@ -0,0 +1,65 @@
+from gladier import GladierBaseClient, GladierBaseTool, generate_flow_definition
+import typing as t
+from pprint import pprint
+
+
+def make_scatter_plot(
+    x: t.List[int], y: t.List[int], name: str = "scatter_plot.png", **data
+):
+    import pathlib
+    import plotly.express as px
+
+    # Make the plot
+    fig = px.scatter(x=x, y=y)
+    fig.write_image(name)
+
+    # Return the filename
+    return str(pathlib.Path(name).absolute())
+
+
+@generate_flow_definition(
+    modifiers={
+        "make_scatter_plot": {
+            "user_endpoint_config": {
+                "container_type": "podman",
+                "container_uri": "ghcr.io/globus-gladier/gladier-client-examples/containers-example:latest",
+            }
+        }
+    },
```

Comment:
V3 allows these modifiers above. We can pass "user_endpoint_config" with our extra options.